doc/classes/RenderingServer.xml

```diff
@@ -3100,6 +3100,20 @@
 				If [code]true[/code], rendering of a viewport's environment is disabled.
 			</description>
 		</method>
+		<method name="viewport_set_fsr_mipmap_bias">
+			<return type="void" />
+			<argument index="0" name="viewport" type="RID" />
+			<argument index="1" name="mipmap_bias" type="float" />
+			<description>
```

Comment:
Affects the final texture sharpness by reading from a lower or higher mipmap. Negative values make textures sharper, while positive values make textures blurrier. When using FSR, AMD recommends you set the mip bias according the to desired quality level with the following preset options "Ultra Quality: -0.38", "Quality: -0.58", "Balanced: -0.79", and "Performance: -1.0".